dashboard/test/controllers/curriculum_tracking_pixel_controller_test.rb

```diff
@@ -0,0 +1,8 @@
+require 'test_helper'
+
+class CurriculumTrackingPixelControllerTest < ActionController::TestCase
+  test "get index" do
+    get :index, params: {from: '/csf-18/pre-express/11/'}
```

Comment:
Is `FirehoseClient` stubbed here already?

Reply:
I don't think so - not in the way I want to stub it for writing tests. 

Do you mean will we log to Firehose every this test runs? No, because Firehose checks if the environment is test or development and skips sending the record if so. (https://github.com/code-dot-org/code-dot-org/blob/cf4bef8351474daecdec0170c4a86a0ba881efef/lib/cdo/firehose.rb#L31)